tools/nodejs/test/udf.test.js

```diff
@@ -9,142 +9,142 @@ describe('UDFs', function() {
         });
 
```

Comment:
No new tests for this functionality?

Reply:
The JS tests are in the arrow extension. The idea being that tests for this functionality should be run in the CI of the extension. 

However, i think it does make sense to have some basic test in the main repo to confirm that the error message that the arrow extension is required is properly shown.

Reply:
Preferably they would be written in typescript to exercise the type stubs as well